erpnext/accounts/doctype/sales_invoice/sales_invoice.json

```diff
@@ -726,7 +725,6 @@
    "read_only": 1
   },
   {
-   "depends_on": "packed_items",
```

Comment:
These seem to be unintentional changes? Pull `develop` branch changes then migrate & add the new status.

Reply:
Yes. Thanks!